### PR TITLE
[docs sprint] Updates docs for events manager

### PR DESCRIPTION
**Pull request by ajar98**

### CODE REVIEW

File: docs/open-source/events-manager.mdx [hunk 1]
```diff
@@ -5,87 +5,60 @@ description: "How events are emitted and consumed."
...
-3. `PHONE_CALL_CONNECTED`: Indicates a phone call has been connected.
-4. `PHONE_CALL_ENDED`: Indicates a phone call has ended.
-5. `RECORDING`: (Vonage Only) Indicates a secure URL containing a recording of the call is available. Requires `recording=true` in `VonageConfig`.
+class CustomEventsManager(EventsManager):
+    def __init__(self):
+        super().__init__([EventType.TRANSCRIPT_COMPLETE])
 
-## Example Usage
+    async def handle_event(self, event: Event):
+        if isinstance(event, TranscriptCompleteEvent):
```
Comment (adnaans) [inline docs/open-source/events-manager.mdx:41]: why do we switch to isinstance here? The `event.type` check is better practice imo

Comment (adnaans) [reply docs/open-source/events-manager.mdx:41]: If it helps for typechecking for coding then fine to leave as-is tho

**Review by adnaans — COMMENTED** [review]
overall looks good just 1 comment

File: docs/open-source/events-manager.mdx (hunk 1 above)
Comment (ajar98) [reply docs/open-source/events-manager.mdx:41]: it's for typechecking

**Review by adnaans — APPROVED** [review]

**Pull request merged**